app/celeryapp/celery_worker.py

```diff
@@ -0,0 +1,10 @@
+from app import celeryapp, create_app
+from config import config_dict
+from app import get_config_mode
+
+app_config = config_dict[get_config_mode.capitalize()]
+
+app = create_app(app_config)
+
+celery = celeryapp.make_celery(app)
```

Comment:
I am somehow  uncomfortable with this naming `celeryapp`
I would configure my celery app in a module called `celery.py` and do all the initializations there

Reply:
I have removed it since the initializing is done in the base app.